addons/a11y/src/register.tsx

```diff
@@ -20,56 +20,56 @@ const PreviewWrapper: FunctionComponent<{}> = p => (
     <Hidden>
       <svg key="svg">
         <defs>
-          <filter id="protanopia">
+          <filter id="Protanopia">
             <feColorMatrix
               in="SourceGraphic"
               type="matrix"
               values="0.567, 0.433, 0, 0, 0 0.558, 0.442, 0, 0, 0 0, 0.242, 0.758, 0, 0 0, 0, 0, 1, 0"
             />
           </filter>
-          <filter id="protanomaly">
+          <filter id="Protanomaly">
```

Comment:
why capitalize?

Reply:
You have a good point, ids are normally lowercase. The id is used to check against the list of the names of the filters which are now capitalized so I switched it. However,  I could just as easily do to lowercase and check against the id